app/frontend/styles/application.scss

```diff
@@ -10,6 +10,7 @@ $govuk-font-url-function: frontend-font-url;
 $govuk-image-url-function: frontend-image-url;
 
 @import "~govuk-frontend/govuk/all";
+@import "~@ministryofjustice/frontend/moj/all";
```

Comment:
There is a downside to doing this: we're importing their full CSS. I don't know how big it is but presumably we only need a small part of it.

Do they allow importing just the CSS for the sub-navigation?

Otherwise, this is fine for now, however I'm going to create a tech debt card to investigate the size of our CSS and introduce something like https://www.purgecss.com to slim it down.

Reply:
Great idea, done & updated!